operator/src/main/java/com/github/streamshub/console/dependents/ConsoleDeployment.java

```diff
@@ -130,4 +136,8 @@ <R extends KubernetesResource> List<R> getResourcesByType(
             Class<R> key) {
         return (List<R>) resources.getOrDefault(key, Collections.emptyList());
     }
+
+    private String pullPolicy(String image) {
+        return image.contains("sha256:") ? "IfNotPresent" : "Always";
+    }
```

Comment:
The idea here is there is no point re-pulling an image that uses a SHA, but a tagged image may have changed and should be re-pulled. Default configuration in the released operator will use image SHAs.